shell/common/rasterizer.cc

```diff
@@ -466,6 +466,17 @@ RasterStatus Rasterizer::DrawToSurface(flutter::LayerTree& layer_tree) {
         raster_status == RasterStatus::kSkipAndRetry) {
       return raster_status;
     }
+    if (shared_engine_block_thread_merging_ && raster_thread_merger_ &&
+        raster_thread_merger_->IsMerged()) {
+      // TODO(73620): Remove when platform views are accounted for.
+      FML_LOG(ERROR)
+          << "Error: Thread merging not implemented for engines with shared "
+             "components.\n\n"
+             "This is likely a result of using platform views with enigne "
+             "groups.  See "
+             "https://github.com/flutter/flutter/issues/73620.";
+      abort();
```

Comment:
I'm a little sad I don't know what the answer is off of the top of my head, but I feel like there was a prior conversation about this at some point and there was a reason why there isn't any `abort()` in our codebase currently. 

b/170074271 vaguely references this but it's largely a reporting issue. Maybe it's ok.

Reply:
`fml::KillProcess`. Its an `abort` with a `noreturn` decoration so the compiler won't be mad about missing return statements from scopes that don't exit.

@xster Are you thinking of `exit`?

Reply:
If there's precedent then LGTM :)